### PR TITLE
wip: Market Build

### PR DESCRIPTION
**Pull request by rynomster**

### CODE REVIEW

File: src/app/market/shared/orders/orders.component.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ export class OrdersComponent implements OnInit {
   constructor(private bid: BidService) {}
 
   ngOnInit() {
-    this.bid.search().subscribe(orders => this.orders = orders )
+    this.bid.search(this.type === 'sell').subscribe(orders => this.orders = orders);
```
Comment (rynomster) [inline src/app/market/shared/orders/orders.component.ts]: This needs to pass the address

File: src/app/core/market/market-state/market-state.service.ts [hunk 2]
```diff
@@ -42,6 +42,7 @@ export class MarketStateService extends StateService implements OnDestroy {
             error => {
               this.stateCallError(method, error, firstError);
               if (timeout) {
+                console.log(error.status);
```
Comment (rynomster) [inline src/app/core/market/market-state/market-state.service.ts]: remove debugging

File: src/app/core/market/api/bid/bid.service.ts [hunk 3]
```diff
@@ -39,9 +39,9 @@ export class BidService {
     });
   }
 
-  search(): Observable<Array<Bid>> {
+  search(mine?: boolean): Observable<Array<Bid>> {
     // Params: flag for buy or sell required
-    const params = ['search'];
+    const params = ['search', "", "", !!mine];
```
Comment (rynomster) [inline src/app/core/market/api/bid/bid.service.ts]: call this with address instead of `mine`

File: src/app/core/market/api/bid/bid.service.ts [hunk 4]
```diff
@@ -39,9 +39,9 @@ export class BidService {
     });
   }
 
-  search(): Observable<Array<Bid>> {
+  search(mine?: boolean): Observable<Array<Bid>> {
```
Comment (rynomster) [inline src/app/core/market/api/bid/bid.service.ts]: parameter for `address` instead of mine

**Comment by anandsinghparihar** [comment]
@AllienWorks could you please  fix some design changes (after refactoring order list component by order-item component css not applied )

**Comment by vikas-cis** [comment]
@AllienWorks I have added the modal for shipping and Escrow , Now you can text or design the modal :)

Thanks

**Pull request merged**